### PR TITLE
adding commodity prices for coal, oil, gas from Ariadne assumptions

### PR DESCRIPTION
**Pull request by toniseibold**
# Adding Ariadne price assumptions for coal, gas and oil

Adding cost assumption from the initial price discussion of the [Ariadne Scenarios](https://docs.google.com/document/d/10V6o-1vPECYu5UiEi4alYS8r-CXqZSRKqJkAGKt1N0Y/edit) summarized in the [table](https://cloud.pik-potsdam.de/index.php/f/19528535).
For calculating prices in 2020Eur/MWh the following sources were taken:

- [Euro2020](https://www.bmz.de/de/ministerium/zahlen-fakten/oda-zahlen/hintergrund/dac-umrechnungskurs-35300)
- 1 bbl = 1.6998 MWh for Brent oil
- 1 t = 8.06 MWh for API2 coal

No values for biogas/-mass, lignite and nuclear

### CODE REVIEW

File: ariadne-data/costs_2020-modifications.csv [hunk 1]
```diff
@@ -1,2 +1,4 @@
 technology,parameter,value,unit,source,further description
-gas,fuel,50.0,EUR/MWh_th,TTF futures,
+gas,fuel,11.2,EUR/MWh_th,Ariadne,
+oil,fuel,22.1982,EUR2020/Mwh,Ariadne,"$2020 = 0.8775 EUR, 1bbl = 1.6998MWh"
```
Comment (fneum) [inline ariadne-data/costs_2020-modifications.csv:3]:
```suggestion
oil,fuel,22.1982,EUR2020/MWh,Ariadne,"$2020 = 0.8775 EUR2020, 1bbl = 1.6998MWh"
```
nit-picking, here and elsewhere

**Review by fneum — COMMENTED** [review]
Unit conversions look right and coincide with the linked Google Doc. Quite low, but I am not the expert. The 2025 price peaks might look funny in the results.

**Review by fneum — COMMENTED** [review]
Also checked with the abovementioned table. Ready to merge when small formatting issues are resolved.

**Comment by nworbmot** [comment]
Looks good to me, fine to merge once @toniseibold makes formatting changes suggested by @fneum.

**Review by fneum — APPROVED** [review]

**Pull request merged**